cityofaustin/atd-vz-data
# Add Law Enforcement Number to the FTT Index

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/11805

For context the law enforcement number is APDs running count of fatal crashes for that year, similar to our "Year to date fatal crash" count. The VZ team will always be inputting it manually.

## Testing
**URL to test:** 
[Netlify](https://deploy-preview-1202--atd-vze-staging.netlify.app/#/fatalities)

**Steps to test:**
1. Click on a crash from the fatalities page
2. The law enforcement number can be found in the Fatalities card 
![image](https://user-images.githubusercontent.com/67130963/229549920-0879d2de-1ac8-484c-bb95-6440f45af944.png)
3. Edit the law enforcement num a few times, it is of type int. Check back at the Fatalities page to see it reflected there.

One thing I was thinking:
~Should the law enforcement number section only be seen on crashes that have fatalities? Or should it stay there for all crashes? Im guessing the VZ team wants it there for all crashes in case something changes but I'm not sure.~

---
#### Ship list
